CSS/CSS.sublime-settings

```diff
@@ -4,11 +4,11 @@
 
     // Controls what scopes default completions will be provided in.
     // Can be a list of strings which are joined before matching.
-    "default_completions_selector": "source.css",
+    "default_completions_selector": "source.css - source.css comment - source.css meta.embedded - source.css source - source.css text",
```

Comment:
I'll need to refresh my memory of multiple negative selector operator precedence to understand what this is doing...
A comment which explains this new default could help :)

Reply:
A reminder was added to settings.

We could also wrap all negative selectors into parens, if that helps improving readability, just like

```
"default_completions_selector": "source.css - (source.css comment | source.css meta.embedded | source.css source | source.css text)",
```

The idea is to exclude CSS comments and all sorts of embedded source/text under source.css.

Current selectors disable completions in `source.css source.php` for instance, but not in `source.php source.css`. Not quite sure it is a realistic example but selectors should just make sure to only exclude stuff under CSS.